R/RunStep.R

```diff
@@ -22,9 +22,9 @@
 #' 2. If a single parameter value is equal to "lData", the full lData object is passed to the function.
 #' 3. If a single parameter value is equal to "lSpec", the full lSpec object is passed to the function.
 #' 4. If a single parameter value is found in names(lMeta), that property is pulled from lMeta (e.g.
-#' lMeta${paramVal}) and passed to the function.
+#' lMeta$\{paramVal\}) and passed to the function.
```

Comment:
![image](https://github.com/user-attachments/assets/6b8fec05-ffba-4cef-ae18-ea9819ccb427)

Is this the intentional "display" of this edit? Also there is a typo at line 15

Reply:
yes, this is the intended display. i'll edit the typo up top. thanks for catching that!